src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj

```diff
@@ -24,43 +24,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU'" />
```

Comment:
Could you please switch the build to reuse the same binaries for both Windows and Unix. We do not need different builds for this anymore.